buildlib/pr/io_demo/az-stage-io-demo.yaml

```diff
@@ -13,6 +13,8 @@ parameters:
   default: 20
 - name: analyzer_allow_list_args
   default: ''
+- name: test_analyzer_args
```

Comment:
maybe make it boolean parameter (type: boolean)
name: analyzer_use_allow_list (or similar)

Reply:
I think that we can use this parameter for send other parameters to analyzer in the future , maybe

Reply:
can we pass the value directly to analyzer, without 'if [[ ${{ parameters.test_analyzer_args }} == '--no-allow-list' ]]; then'?